### PR TITLE
Add Elasticsearch service

### PR DESCRIPTION
**Pull request by jhettler**
Add Elasticsearch to docker-compose. Users are able to get demo with search working.

### CODE REVIEW

**Comment by jinhyukchang** [comment]
Thanks @jhettler !
Could you take a look at CLA (Contributor License Agreement) by clicking detail? Once this one is passed, I will approve and merge the change.

**Comment by jhettler** [comment]
Hi @jinhyukchang, I have done it, unfortunately the check is still not passed. Any ideas, how to rerun CLA check? 

Thanks!

**Comment by feng-tao** [comment]
@jhettler talked with our open source team. The error is due to ```An author in one of the commits has no associated github name```.

I think you need to config your github user.name and user.email locally first(could confirm with git config --list), then push to the branch. Let us know if it works.

**Comment by jinhyukchang** [comment]
@jhettler You may need to amend the commit and force push. Let us know. Thanks!

**Comment by jhettler** [comment]
Sorry, I didn't realize that I am pushing changes from company laptop where I a have global git config to my company email. Looks good now.

**Comment by feng-tao** [comment]
thanks @jhettler for the contribution.

**Comment by amweiss** [comment]
I'm not sure if this was intentional, but it removed
```
      networks:
        - amundsennet
```
from the neo4j container so it's not accessible from the other containers anymore.

**Comment by feng-tao** [comment]
thanks for reporting @amweiss . Let me take a look.
cc @jinhyukchang @jhettler as well.

**Comment by feng-tao** [comment]
@amweiss , it seems that it is still there(https://github.com/lyft/amundsenfrontendlibrary/blob/master/docker-amundsen.yml)?

**Comment by amweiss** [comment]
@feng-tao it was removed from the neo4j entry and put into the new elasticsearch one.
image

**Comment by feng-tao** [comment]
thanks @amweiss for pointing it out. You are right :)

**Comment by feng-tao** [comment]
will prepare a pr shortly.

File: docker-amundsen.yml [hunk 1]
```diff
@@ -15,13 +15,20 @@ services:
           - 7687:7687
       volumes:
           - ./example/docker/neo4j/conf:/conf
+  elasticsearch:
+      image: elasticsearch:6.7.0
+      container_name: es_amundsen
+      ports:
+          - 9200:9200
       networks:
```
Comment (amweiss) [inline docker-amundsen.yml]: @feng-tao this is the line I was referencing, it's hard to point out but the diff shows it present in the neo4j section, but then the elasticsearch entry is added in between so it assigns the network to elasticsearch but not in neo4j anymore.

Comment (feng-tao) [reply docker-amundsen.yml]: thanks @amweiss for reporting. Just commit the change in master :)

**Comment by jhettler** [comment]
Sorry guys, I was preparing the environment for running without docker-machine, so I removed all networks and if I commited it, it put it back and I just checked the diff and it was not obvious that there should be this deletion from diff. Thanks for fix.

**Comment by feng-tao** [comment]
@jhettler  no worry :)

**Pull request merged**